main.gms

```diff
@@ -260,7 +260,7 @@ $setGlobal initialCap  on             !! def = on
 *'---------------------    11_aerosols    --------------------------------------
 *'
 *' * (exoGAINS):
-$setGlobal aerosols  exoGAINS         !! def = exoGAINS
+$setGlobal aerosols  exoGAINS2025         !! def = exoGAINS2025
```

Comment:
please describe both realizations for the transition periode where we have both available